### PR TITLE
feat(console): multi-text-input delete reminder

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

feat(console): multi-text-input delete reminder

Refer to the design doc:
image
https://www.figma.com/file/iroUuWbeZ9fqcqXNwqYZai/Admin-Console-UI?node-id=2371%3A51066

## Linear Issue Reference

* LOG-2306

## Testing

multi-text-reminder

image

### CODE REVIEW

File: packages/console/src/components/MultiTextInput/DeletionReminder.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import ModalLayout from '../ModalLayout';
+import * as styles from './index.module.scss';
+
+type Props = {
+  title: I18nKey;
+  onConfirm: () => void;
+  onClose: () => void;
+};
+
+const DeletionReminder = ({ title, onConfirm, onClose }: Props) => {
```
Comment (demonzoo) [inline packages/console/src/components/MultiTextInput/DeletionReminder.tsx:15]: nit: `DeleteConfirm` would be slightly better

File: packages/console/src/components/MultiTextInput/DeletionReminder.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  const { t } = useTranslation();
+
+  return (
+    <ModalLayout
+      title="form.reminder"
```
Comment (demonzoo) [inline packages/console/src/components/MultiTextInput/DeletionReminder.tsx:20]: form.confirm

File: packages/console/src/components/MultiTextInput/index.tsx [hunk 3]
```diff
@@ -55,10 +63,15 @@ const MultiTextInput = ({ value, onChange, error }: Props) => {
...
+            {fieldIndex > 0 && (
               <IconButton
                 onClick={() => {
-                  handleRemove(fieldIndex);
+                  if (fieldValue.trim().length === 0) {
+                    handleRemove(fieldIndex);
+
+                    return;
+                  }
+                  setDeleteFieldIndex(fieldIndex);
```
Comment (demonzoo) [inline packages/console/src/components/MultiTextInput/index.tsx:74]: use `if else` would be better?

File: packages/console/src/components/MultiTextInput/index.tsx [hunk 4]
```diff
@@ -1,25 +1,33 @@
...
   error?: MultiTextInputError;
 };
 
-const MultiTextInput = ({ value, onChange, error }: Props) => {
+const MultiTextInput = ({ title, value, onChange, error }: Props) => {
   const { t } = useTranslation(undefined, {
     keyPrefix: 'admin_console',
   });
 
+  const [deleteFieldIndex, setDeleteFieldIndex] = useState<number | undefined>();
```
Comment (demonzoo) [inline packages/console/src/components/MultiTextInput/index.tsx:29]: nit: just `useState<number>()` would be good

**Review by demonzoo — APPROVED** [review]

File: packages/console/src/components/MultiTextInput/index.tsx (hunk 4 above)
Comment (xiaoyijun) [reply packages/console/src/components/MultiTextInput/index.tsx]: updated

**Pull request merged**